src/main/java/org/sagebionetworks/web/client/presenter/users/RegisterAccountPresenter.java

```diff
@@ -1,112 +1,36 @@
 import com.google.gwt.user.client.ui.AcceptsOneWidget;
 import com.google.inject.Inject;
-import org.sagebionetworks.repo.model.principal.AliasType;
-import org.sagebionetworks.web.client.ClientProperties;
-import org.sagebionetworks.web.client.DisplayConstants;
-import org.sagebionetworks.web.client.GlobalApplicationState;
-import org.sagebionetworks.web.client.SynapseClientAsync;
-import org.sagebionetworks.web.client.place.Profile;
 import org.sagebionetworks.web.client.place.users.RegisterAccount;
 import org.sagebionetworks.web.client.presenter.Presenter;
-import org.sagebionetworks.web.client.security.AuthenticationController;
-import org.sagebionetworks.web.client.view.users.RegisterAccountView;
-import org.sagebionetworks.web.client.view.users.RegisterWidget;
-import org.sagebionetworks.web.client.widget.entity.controller.SynapseAlert;
+import org.sagebionetworks.web.shared.WebConstants;
 
 public class RegisterAccountPresenter
   extends AbstractActivity
-  implements RegisterAccountView.Presenter, Presenter<RegisterAccount> {
-
-  private RegisterAccount place;
-  private RegisterAccountView view;
-
-  private RegisterWidget registerWidget;
-  private AuthenticationController authController;
-  private GlobalApplicationState globalAppState;
-  SynapseAlert googleSynAlert;
-  private SynapseClientAsync synapseClient;
+  implements Presenter<RegisterAccount> {
 
   @Inject
-  public RegisterAccountPresenter(
-    RegisterAccountView view,
-    SynapseClientAsync synapseClient,
-    RegisterWidget registerWidget,
-    AuthenticationController authController,
-    GlobalApplicationState globalAppState,
-    SynapseAlert googleSynAlert
-  ) {
-    this.view = view;
-    this.registerWidget = registerWidget;
-    this.authController = authController;
-    this.globalAppState = globalAppState;
-    this.synapseClient = synapseClient;
-    this.googleSynAlert = googleSynAlert;
-    fixServiceEntryPoint(synapseClient);
-    view.setRegisterWidget(registerWidget.asWidget());
-    view.setGoogleSynAlert(googleSynAlert);
-    registerWidget.setSentEmailCallback(() -> {
-      view.showEmailSentUI(true);
-    });
-  }
+  public RegisterAccountPresenter() {}
 
   @Override
-  public void start(AcceptsOneWidget panel, EventBus eventBus) {
-    // Install the view
-    panel.setWidget(view);
-  }
+  public void start(AcceptsOneWidget panel, EventBus eventBus) {}
 
   @Override
   public void setPlace(RegisterAccount place) {
-    this.place = place;
     String token = place.toToken();
-    String email = "";
-    if (token != null && !ClientProperties.DEFAULT_PLACE_TOKEN.equals(token)) {
-      email = token.trim();
+    if (token != null && token.contains("@")) {
+      // is likely an email address
+      Window.Location.replace(
+        WebConstants.ONESAGE_PRODUCTION_URL + "/register1?email=" + token
+      );
+    } else {
+      Window.Location.replace(
+        WebConstants.ONESAGE_PRODUCTION_URL + "/register1"
+      );
     }
```

Comment:
Redirect, if a token was included, forward the token to OneSage

Reply:
This is always URL decoded, right?  (We don't need to look for "%40" as well?)